src/main/java/org/c4sg/mapper/OrganizationMapper.java

```diff
@@ -17,4 +22,11 @@ public Organization getOrganizationEntityFromDto(OrganizationDTO organizationDTO
 		Organization organization = map(organizationDTO, Organization.class);
 		return organization;
 	}
+	
+	public OrganizationDTO getOrganizationDtoFromEntity(UserOrganization userOrganization){
```

Comment:
Instead of create this method, what about reuse this method: 
`public OrganizationDTO getOrganizationDtoFromEntity(Organization organization){}`

and pass this `userOrganization.getOrganization()` as argument.